Oneirocom/Magick
# New AIDE navbar

## What Changed:

I have added the new navbar for the sidebar,  I just added the menu icon and when it is clicked it opens the menus and it's submenus too for each menu

We are displaying them conditionally if the feature flag is true we display  the new navbar else old navbar

## How to test:

- Navigate to the test-navbar branch, click on the menu icon then check the pop up menus 

## Additional information:


https://github.com/Oneirocom/Magick/assets/55635977/b83033e9-5f53-44a4-93de-5a2ba76f3742
